packages/two_dimensional_scrollables/lib/src/table_view/table.dart

```diff
@@ -49,8 +49,10 @@ import 'table_span.dart';
 /// ```dart
 /// TableView.builder(
 ///   cellBuilder: (BuildContext context, TableVicinity vicinity) {
-///     return Center(
-///       child: Text('Cell ${vicinity.column} : ${vicinity.row}'),
+///     return TableViewCell(
```

Comment:
There isn't a way to analyze snippets in this repo, right?

Reply:
Not that I am aware of. I searched the repo for cases of this but did not find any.